arangosh/Backup/BackupFeature.cpp

```diff
@@ -681,68 +681,73 @@ void BackupFeature::collectOptions(std::shared_ptr<options::ProgramOptions> opti
                      static_cast<std::underlying_type<Flags>::type>(Flags::Hidden));
 
   options->addOption("--allow-inconsistent",
-                     "whether to attempt to continue in face of errors (may "
-                     "result in inconsistent backup state)",
+                     "whether to attempt to continue in face of errors. "
+                     "May result in inconsistent backup state (create operation)",
                      new BooleanParameter(&_options.force));
 
-  options->addOption("--identifier", "a unique identifier for a backup",
+  options->addOption("--identifier",
+                     "a unique identifier for a backup "
+                     "(restore/upload/download operation)",
                      new StringParameter(&_options.identifier));
 
-  //  options->addOption("--include-search", "whether to include ArangoSearch data",
+  //  options->addOption("--include-search",
+  //                     "whether to include ArangoSearch data (??? operation)",
```

Comment:
Does not matter, is not implemented, and we will have to revisit once we can backup ArangoSearch data.